cluster/charts/universal-crossplane/templates/crossplane/rbac-manager-deployment.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
     app: {{ template "crossplane.name" . }}-rbac-manager
     release: {{ .Release.Name }}
     {{- include "crossplane.labels" . | indent 4 }}
+  {{- with .Values.customAnnotations }}
```

Comment:
@pierluigilenoci thanks for the PR. 

We are inheriting the helm chart manifests from upstream, so any changes we made here would be overridden with `make generate`. Would you mind opening against crossplane/crossplane changing the following file? https://github.com/crossplane/crossplane/blob/master/cluster/charts/crossplane/templates/rbac-manager-deployment.yaml#L10